config/services.php

```diff
@@ -31,4 +31,8 @@
         'region' => env('AWS_DEFAULT_REGION', 'us-east-1'),
     ],
 
+    'telegram-bot-api' => [
```

Comment:
Since telegram is the service we should keep to the standard as referencing the service by name so in this case can you update `telegram-bot-api` to just `telegram` and then change the `token` to `bot_token` so we maintain that distinction?

Reply:
Nevermind on this one, saw you used the docs as intended.

Reply:
That's correct, just follow the full docs...